plugins/lookup/onepassword_doc.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: onepassword_doc
+    author:
+      - Sam Doran (@samdoran)
+    requirements:
+      - C(op) 1Password command line utility. See U(https://support.1password.com/command-line/)
```

Comment:
Can this also work with v1? The code looks like it requires v2 to me, but it might only need things from `OnePassCLIv2` that also work with v1.

Reply:
It does not work with v1. I'm not sure if docs were supported in v1 and I'd rather only support v2 going forward.

Reply:
In that case I would mention in this line that v2 only is supported :)